pkg/cli/testing.go

```diff
@@ -34,11 +34,11 @@ func (fakeClient) RootURI() string {
 	return unitTestServiceURL + "/v1"
 }
 
-func (fakeClient) HostService(host string) hoclient.HostOrchestratorService {
+func (fakeClient) HostService(host string) (hoclient.HostOrchestratorService, error) {
 	if host == "" {
 		panic("empty host")
```

Comment:
Probably you want to return an error than occurring panic?

Reply:
Done.